### ISSUE TITLE
Clean up result labels

### ISSUE THREAD

**Issue by joebayles** [Esri/visibility-addin-dotnet#161]
![image](https://cloud.githubusercontent.com/assets/5016557/22051388/a5b9b6f0-dcf7-11e6-811a-2a60d2ca28ae.png)

**For LLOS Targets:**
- [ ] Capitalize visible
- [x] Change "Can be seen" to "Visible"

**For LLOS Output**
- [x] Capitalize visible

**For LLOS Sight Lines**
- [x] Capitalize visible

**For RLOS Visibility**
- [x] Change "non visible data" to "Not Visible"

**If possible, I would remove All other values** 
- [ ] LLOS Observers - This will be addressed in another issue

**Remove the grouping** 
- [x] LLOS Targets
- [ ]  LLOS Observers
- [x]  LLOS Output
- [x] LLOS Sight Lines

**Comment by topowright** [comment]
Updates addressed in PR: 
https://github.com/Esri/visibility-addin-dotnet/pull/255

I was not able to remove the all other values from the LLOS_Observers
![image](https://user-images.githubusercontent.com/5488313/35465584-e7217fb6-02b1-11e8-8017-3f0a5f1c922f.png)

**Comment by topowright** [comment]
This needs to be verified once PR #255 is merged.  @dfoll @ACueva

**Comment by topowright** [comment]
This is now ready to be verified.

**Comment by topowright** [comment]
Closing this issue based on the work that has been done to this point.
